man/fi_mr.3.md

```diff
@@ -736,6 +743,10 @@ The follow flag may be specified to any memory registration call.
   API AllocHost call (as opposed to using malloc-like host allocation,
   unified/shared memory allocation, or AllocDevice).
 
+*FI_HMEM_DATA_GDR_HANDLE*
+: This flag indicates that hmem_data points to a GDR handle. The owner of the
+handle is responsible for unsetting this flag if the handle is invalidated.
```

Comment:
Remove comment about 'owner of the handle'.  This man page is user-facing, and that comment doesn't make sense.

Reply:
I will remove this for now. My intention is to define the expectation if hmem_data is passed from the application, such that libfabric does not own the handle.

Reply:
>  Having the same value and being an alias are not the same. Flags for mr_attr should go into fi_domain.h. The FI_HMEM_HOST_ALLOC and FI_HMEM_DEVICE_ONLY flags should have gone there (and probably have been given different values). See comment in fi_domain.h on memory registration flags. I would not choose a value that conflicts with FI_HMEM. The possible need to use that flag is too high.
In general, flags for functions typically start at bit 59 and go down from there, skipping any conflicts (mostly with caps). So, I'd make this bit 59, since that's still available.

I see. Thanks for the tip. I will include a commit to move the flags.